frappe_private_comment/public/js/controls/timeline.js

```diff
@@ -1,3 +1,5 @@
+frappe.require(["/assets/frappe_private_comment/js/controls/replies.js"]);
```

Comment:
Why is this line required here? We do have this employee in `frappe_private_comment.js` file

Reply:
Discussed, the imported function was not working without this import.